src/tests/RegistrationTest.test.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @jest-environment node
+ */
+
+import Registration from './../components/Registration/Registration';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import AuthService from 'services/AuthService';
+
+describe('Test case for testing registration', () => {
+	let wrapper;
+	test('email check', () => {
+		wrapper = shallow(<Registration />);
+		wrapper.find('input[type="text"]').simulate('change', { target: { name: 'email', value: 'johndoe@gmail.com' } });
+		expect(toJson(wrapper.state('email'))).toEqual('johndoe@gmail.com');
+	})
+
+	it('first name check', () => {
+		wrapper = shallow(<Registration />);
+		wrapper.find('input[type="text"]').simulate('change', { target: { name: 'firstName', value: 'John' } });
+		expect(toJson(wrapper.state('firstName'))).toEqual('John');
+	})
+
+	it('last name check', () => {
+		wrapper = shallow(<Registration />);
+		wrapper.find('input[type="text"]').simulate('change', { target: { name: 'lastName', value: 'Doe' } });
+		expect(toJson(wrapper.state('lastName'))).toEqual('Doe');
+	})
+
+	it('password check', () => {
+		wrapper = shallow(<Registration />);
+		wrapper.find('input[type="password"]').simulate('change', { target: { name: 'password', value: '123456' } });
+		expect(toJson(wrapper.state('password'))).toEqual('123456');
+	})
+
+	it('login check', async () => {
```

Comment:
Registration check? Also, I wouldn't want to depend on the backend running in order for a frontend unit test to work. You should either mock this or remove it - it isn't crucial.